packages/components/src/components/domainproperties/DomainForm.tsx

```diff
@@ -922,6 +923,10 @@ export class DomainFormImpl extends React.PureComponent<IDomainFormInput, IDomai
                                 {importDataChildRenderer && importDataChildRenderer()}
                             </ImportDataFilePreview>
                         )}
+                        {filePreviewData &&
```

Comment:
Do you think we should instead of including ColumnMapping inside ImportDataFilePreview keep it above ImportDataFilePreview? That way, it will be consistent between the shared and non shared studies.

Reply:
Yeah, I think that is a good idea. If you don't mind changing that, I think it would make more sense to have the layout be: fields listing, column mapping, file preview and import option. Then in the no-file case, it can just be the fields listing and in the dataspace case it can be fields listing plus column mapping. Good suggestion.